packages/cactus-plugin-cc-tx-visualization/src/main/typescript/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./public-api";
```

Comment:
@RafaelAPB Could you please also add an index.web.ts to match the common export structure of the other packages?

Reply:
done!